sway/desktop/output.c

```diff
@@ -376,6 +475,20 @@ static void render_container_simple_border_pixel(struct sway_output *output,
 			output->wlr_output->transform_matrix);
 }
 
+/**
+ * Render decorations for a tabbed view with "border normal".
+ */
+
+static void render_container_tabbed_border_normal(struct sway_output *output,
+		struct sway_container *con, struct wlr_texture **title_texture,
+		int depth, bool active) {
```

Comment:
Why not pass a pointer to the appropriate border_colors struct like the layout does? Remember, some tabs might need to be rendered with the urgent class.